### PR TITLE
Fix failing test (Issue #11)

### PR DESCRIPTION
**Pull request by devcamcar**

### CODE REVIEW

File: test/logstash/snippets/vcap-spec.rb [hunk 1]
```diff
@@ -69,7 +69,7 @@
     sample("@type" => "relp", "host" => "1.2.3.4", "@message" => '<78>Apr 24 04:03:06 localhost crontab[32185]: (root) LIST (root)') do
       insist { subject["tags"] } == [ 'syslog_standard' ]
       insist { subject["@type"] } == 'relp'
-      insist { subject["@timestamp"] } == Time.iso8601("2014-04-24T04:03:06.000Z")
+      insist { subject["@timestamp"] } == Time.iso8601("2015-04-24T04:03:06.000Z")
```
Comment (mrdavidlaing) [inline test/logstash/snippets/vcap-spec.rb]: @devcamcar - thanks!

however, would this fix not also fail next year?

Could you update this PR so it checks for `Time.iso8601("#{current_year}-04-24T04:03:06.000Z")` please

**Comment by devcamcar** [comment]
@mrdavidlaing Updated to not self destruct in a year :)

**Comment by mrdavidlaing** [comment]
@devcamcar - Thanks!

**Comment by mrdavidlaing** [comment]
@devcamcar,

Devin,

Sorry to trouble you; but would you mind sending me a signed CLA for the contribution you made to the Logsearch for CloudFoundry project - https://github.com/logsearch/logsearch-for-cloudfoundry/pull/12 - back in March.

I am just trying to wrap up the legals around contributions so we can take on more corporate sponsorship; and noticed that I don't have a CLA on file for you.

Should be as simple as printing & scanning (or attaching an electronic picture of your signature) for https://github.com/logsearch/licensing/blob/master/contributor-license-agreements/LogSearch_Individual_Contributor_License_Agreement.md and emailing it to me at david@stayup.io

Thanks!

David

**Pull request merged**